data/pilots/first-order/tie-fo-fighter.json

```diff
@@ -42,16 +42,15 @@
       "caption": "Omega Leader",
       "initiative": 6,
       "limited": 1,
-      "cost": 4,
-      "loadout": 15,
+      "cost": 3,
+      "loadout": 7,
       "xws": "midnight",
       "ability": "While you defend or perform an attack, if you have a lock on the enemy ship, that ship's dice cannot be modified.",
       "image": "https://infinitearenas.com/xw2/images/pilots/midnight.png",
       "artwork": "https://infinitearenas.com/xw2/images/artwork/pilots/midnight.png",
       "slots": [
         "Talent",
         "Talent",
-        "Missile",
         "Modification",
         "Modification",
         "Tech"
```

Comment:
This is certainly not worth doing in this PR, and probably never worth doing, but maybe some day I'll be annoyed enough about something I have at work to do it - I wish that the pilots in these files and the orders of their slots were the same as the points doc. I realize that tehy probalby WERE before, but now they're not, and I suspect makes error non-trivially more likely to occur.

Reply:
> This is certainly not worth doing in this PR, and probably never worth doing, but maybe some day I'll be annoyed enough about something I have at work to do it - I wish that the pilots in these files and the orders of their slots were the same as the points doc. I realize that tehy probalby WERE before, but now they're not, and I suspect makes error non-trivially more likely to occur.

Could not possibly agree more. To make matters worse, the script attempts to reformat upgrades seeming randomly either compressing the "slots" name to a single line, or expanding it over multiple lines. But yes, extremely agree, I think that effort would be very much worth it.